freiburg/projects.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+---
+
+# Our projects
+
+## Galaxy and its community
+
+The Freiburg Galaxy team is highly involved in many projects of the Galaxy community:
+
+- Galaxy core development and maintenance
+- Tool integration into Galaxy and their maintenance through
+    - [Intergalactic Utilities Commission](https://galaxyproject.org/iuc/)
+    - [Björn's repository](https://github.com/bgruening/galaxytools)
+- [Galaxy Training Material](https://galaxyproject.github.io/training-material/), a collection of Galaxy-related tutorials and its infrastructure, developed and maintained by the community
+- [Galaxy Docker](https://github.com/bgruening/docker-galaxy-stable) to provide an easy distributable full-fledged Galaxy installation
+- Development and maintenance of different Galaxy flavours
+    - [RNA-Workbench](http://bgruening.github.io/galaxy-rna-workbench/) 
+    - [ASaiM](http://asaim.readthedocs.io/en/latest/) for metagenomics
+- Jupyter integration into Galaxy
+
+## Bioconda and Biocontainers
+
+Spending valuable time in compilation/installation hell should not be a problem nowadays. Therefore, the Freiburg Galaxy team is leading the Bioconda ([http://bioconda.github.io](http://bioconda.github.io)) and BioContainers ([https://biocontainers.pro](https://biocontainers.pro)) projects, which together form the best stack for reproducible science ever.
+
+Bioconda is a channel for the conda package manager specializing in bioinformatics with more than 3,000 bioinformatics packages ready to use with `conda install`. BioContainers is an open source and community-driven framework which provides system-agnostic executable environments for bioinformatics software, with more than 3,000 containers ready to use.
+
+## HiCExplorer
+
+[HiCExplorer](https://hicexplorer.readthedocs.io/en/documentation/) is a set of programs to process, normalize, analyze and visualize Hi-C data. It addresses the common tasks of Hi-C analysis from processing to visualization.
+
+The Freiburg Galaxy team is involved in the maintenance and new development of HiCExplorer, and particularly its integration into Galaxy: [https://hicexplorer.usegalaxy.eu/](https://hicexplorer.usegalaxy.eu/)
+
+## de.NBI and ELIXIR
+
+The Freiburg Galaxy team is part of [de.NBI](http://www.denbi.de/), the German Network for Bioinformatics Infrastructure, through the [RNA Bioinformatics Center](http://www.denbi.de/rbc). Via de.NBI, the Freiburg Galaxy team is also involved in [ELIXIR](https://www.elixir-europe.org), as technical coordinator for the the [German node](https://www.elixir-europe.org/about-us/who-we-are/nodes/germany).
+
+The Freiburg Galaxy team is also co-leading the [ELIXIR Galaxy Community
+](https://www.elixir-europe.org/communities/galaxy), particularly with the maintenance of European Galaxy instance: [usegalaxy.eu](https://usegalaxy.eu/).
+
+## Neuromac? Single-cell?
```

Comment:
I'm not sure we should add data-analysis projects, thats hard to keep track. We should probably use the SFB terms here ... if we want to mention those.

Reply:
I agree to Björn